tests/test_builtin_casters.py

```diff
@@ -299,7 +299,7 @@ def cant_convert(v):
     assert noconvert(7) == 7
     cant_convert(3.14159)
     # TODO: Avoid DeprecationWarning in `PyLong_AsLong` (and similar)
-    if (3, 8) <= env.PY < (3, 10):
+    if (3, 8) <= env.PY < (3, 10) and env.CPYTHON:
```

Comment:
TODO here and below, to remove the `and env.CPYTHON` after the 3.11 `co_varnames` issue is resolved?

Reply:
This is a PyPy todo, pypy38 didn't adapt to the changes in CPython.